spot-contracts/contracts/FeePolicy.sol

```diff
@@ -136,6 +108,21 @@ contract FeePolicy is IFeePolicy, OwnableUpgradeable {
     /// @notice Lower and upper fee percentages for flash redemption.
     Range public flashRedeemFeePercs;
 
+    //-----------------------------------------------------------------------------
+    // Incentive parameters
+
+    /// @notice The magnitude of perp's daily debasement (when dr <= 1).
+    uint256 public debasementValueChangePerc;
```

Comment:
These are maximum values, right?

How about naming like--
`maxPerpDebasePerc` and `maxPerpEnrichPerc`

(But see the other comment about whether we need these or not)